src/test/util.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+export function getFlowingState(stream: NodeJS.ReadableStream): boolean {
+  // Cast our streams to <any> so that we can check the flowing state.
+  // _readableState is undocumented in the Node.js TypeScript definition,
+  // however it is the supported way to assert if a stream is flowing or not.
+  // See: https://nodejs.org/api/stream.html#stream_three_states
+  const privateReadableState = (<any>stream)._readableState;
+  return privateReadableState.flowing;
```

Comment:
Nice to factor this out, but I'd go even a step further and make it more of an assertion in itself:
 * Rename to `isFlowing`
 * Use it in a plain assert: `assert(isFlowing(stream))`
 * Since`privateReadableState.flowing` is null sometimes: `return privateReadableState.flowing == true`

This is short of making an actual Chai plugin, but that's a lot more work